lms/djangoapps/instructor/views/api.py

```diff
@@ -1198,15 +1199,15 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
 
     available_features = instructor_analytics.basic.AVAILABLE_FEATURES
 
-    # Allow for microsites to be able to define additional columns.
+    # Allow for sites to be able to define additional columns.
```

Comment:
Actually this was in the master but not in my previous commit which was old which I had cherry-picked.